content/ADR-125-realm-modifiers-for-worlds-in-renderer.md

```diff
@@ -0,0 +1,196 @@
+
+Discoverability of content will be outside-in, as content creators will be able to share easy to remember URLs to enter the World directly, or inside-out using a join command. Anybody will be able to join a World.
+
+
+## Solution Space Exploration
+
+To be able to modify the features necessary in each world, we are developing a Realms Plugin which contains Realms Modifiers. Each of this modifiers will have the responsibility to alter DCL functionality by checking if we are entering a user-generated realm or a DAO realm. 
+
+## Specification
+
+### Realms Plugin
+
+The realms plugin is the entry point for the realms modifier. Its is created under the `explorer-realms_modifier_plugin` flag. It's the bridge which receives the 'AboutResponse' sent by kernel and distributes it among all implemented modifiers. By comparing the just accessed realms with the list of available catalysts, it determines if a realm is part of the DAO or a user created world. This variable is necessary for modifiers to execute certain functionaly. On the near future, we could add this bool to the 'AboutResponse' section, so this responsability relies complete on the data and not on a comparison.
+
+### Realms Modifiers Interface
+
+Describes the behaviour necessesary for realms modifiers. Its current implementation requires to specify if we are on a DAO realm; since some behaviour still relies on this property
+
+```
+namespace DCLPlugins.RealmsPlugin
+{
+    public interface IRealmsModifier : IDisposable
+    {
+        void OnEnteredRealm(bool isCatalyst, AboutResponse realmConfiguration);
+    }
+}
+```
+
+ The current modifiers currently implemented area:
+
+- **Realms Blocker Modifier**
+
+Controls how the green blockers should be shown on each realm. If a realm is a Catalyst, or it has a value on Configurations.CityLoaderContentServer; then the green blockers will be shown.
+
+- **Realms Minimap Modifier**
+
+Controls the functionality associated to the minimap when entering a Realm. It determines if the minimap should be visible using the realmConfiguration.Configurations.Minimap.Enabled value. Also, it determines if the Jump To Home Button should be present. This button will appear on any given user world, and it will take the user back to the home location in the most populated catalyst. 
+When needed, this modifier will implement the change of the minimap image; as a user generated world will have the possibility to show its own minimap png.
+	
+
+### Data Structure 
+
+`AboutResponse` contains the info of the realm which has just been accessed. This class is autogenerated according to protobuff specifications. Each modifier gets access to the AboutResponse, and gets the info necessary to execute its behaviour. It has the following structure: 
+
+```typescript
+type About = {
```

Comment:
This is duplicated info, maybe reference ADR-110 for clarity? https://adr.decentraland.org/adr/ADR-110